packages/@storefront/flux-capacitor/src/core/requests/utils.ts

```diff
@@ -182,6 +182,11 @@ namespace RequestHelpers {
 
   export const chain = <T>(...fns: Array<(...obj: any[]) => T>): T =>
     fns.reduce((final, fn) => fn(final) || final, <T>{});
+
+  export const attachSessionId = <T>(state: Store.State): ((r: T) => T&{ sessionId: Store.SessionId }) => {
```

Comment:
re: `T&{ ...`, do we normally include a space between the `&` operator and operands? If so, let's do that here.

Reply:
Will do.